srcpkgs/samba/template

```diff
@@ -11,7 +11,7 @@ _auth_modules="auth_unix,auth_wbc,auth_server,auth_netlogind,autH_script,auth_sa
 configure_args="--enable-fhs --sbindir=/usr/bin
  --localstatedir=/var --sysconfdir=/etc --with-piddir=/run/samba
  --with-sockets-dir=/run/samba --with-lockdir=/run/lock/samba
- --with-privatedir=/etc/samba/private --with-pammodulesdir=/usr/lib/security
```

Comment:
Changing privatedir will break all existing samba installation, since it stores things like user accounts.